Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Builder stage
+FROM openjdk:17-jdk-alpine as builder
+WORKDIR online-book-store
+ARG JAR_FILE=target/*.jar
+COPY ${JAR_FILE} online-book-store.jar
+RUN java -Djarmode=layertools -jar online-book-store.jar extract
+
+# Final stage
+FROM openjdk:17-jdk-alpine
+WORKDIR application
+COPY --from=builder online-book-store/dependencies/ ./
+COPY --from=builder online-book-store/spring-boot-loader/ ./
+COPY --from=builder online-book-store/snapshot-dependencies/ ./
+COPY --from=builder online-book-store/application/ ./
```

Comment:
I don't see any `application` folder at builder stage

or it is one of default folders after jar extraction?